subiquity/server/controllers/install.py

```diff
@@ -426,10 +427,32 @@ async def get_target_packages(self, context):
         name="install_{package}",
         description="installing {package}")
     async def install_package(self, *, context, package):
-        await run_curtin_command(
-            self.app, context, 'system-install', '-t', self.tpath(),
-            '--', package,
-            private_mounts=False)
+        """ Attempt to download the package up-to three times, then install it.
+        """
+        for attempt, i in enumerate(reversed(range(3))):
```

Comment:
this line bugs me, it doesn't scan.  I think it's the mix of using `i` alongside the other.  I'd prefer a clearer name on `i` - maybe `attempts_remaining` or something, or to reduce this to a single variable.

Reply:
yep, this is definitely hard on the eyes.. I really want to come up with a retry mechanism we can use at various places, but haven't come up with the right design for it yet.